website/src/content/main-concepts/user-permissions.mdx

```diff
@@ -2,10 +2,212 @@
+
 <Info>
 
-See [User Permission Concepts](https://docs.commercetools.com/merchant-center/user-permissions.html).
+A manage access always implies view access as well.
 
 </Info>
 
-_Coming soon_
+A general permission is represented as a **string**, with the resource name prefixed by `View` or `Manage`. The following resource names are currently supported:
+
+- `Products`
+- `Categories`
+- `Customers`
+- `CustomerGroups`
+- `Orders`
+- `ProductDiscounts`
+- `CartDiscounts`
+- `DiscountCodes`
+- `ProjectSettings`
+- `ProductTypes`
+- `DeveloperSettings`
+
+Some examples are: `ViewProducts`, `ManageOrders`, etc.
+
+## Action rights
+
+Action rights can be used to refine the general permissions. For example, manage access to products can be limited to exclude publishing.
+
+An action right is represented as an object with the shape `{ group: string, name: string }`. The `group` relates to the permission while the `name` is the action right itself. Currently the following action rights for the `group: 'products'` exist:
+
+- `PublishProducts`
+- `UnpublishProducts`
+- `AddPrices`
+- `EditPrices`
+- `DeletePrices`
+- `DeleteProducts`
+- `AddProducts`
+
+# Using configured permissions within a Custom Application
+
+The [`@commercetools-frontend/permissions`](https://github.com/commercetools/merchant-center-application-kit/blob/master/packages/permissions/README.md) package should be used to implement and apply the user permissions for the specific parts of the Custom Applications. The package exposes React components and React hooks to make it easy to work with it:
+
+- `useIsAuthorized()`
+- `injectAuthorized()`
+- `<RestrictedByPermissions>`
+
+## `useIsAuthorized(options)`
+
+A React hook that evaluates the demanded permissions against the user permissions and returns a `boolean` value. Only the demanded permissions need to be explicitly provided, the user permissions are implicitly read from the application context.
+
+### Supported named arguments
+
+- `demandedPermissions`: an array of [general permissions](#general-permissions).
+- `demandedAtionRights`: (_optional_) an array of [action rights](#action-rights).
+
+### Usage
+
+```jsx
+import { PERMISSIONS } from './constants/permissions.js';
+
+const ExampleComponent = () => {
+  const canManageProducts = useIsAuthorized({
+    demandedPermissions: [PERMISSIONS.ManageProducts],
+  });
+
+  return <span>{ canManageProducts ? 'Authorized' : 'Not authorized' }</span>;
+};
+```
+
+## `injectAuthorized(permissions, [options])`
+
+A React higher-order component that evaluates the demanded permissions against the user permissions and injects the result as a component prop (default to `isAuthorized`).
+
+### Supported arguments
+
+- `demandedPermissions`: an array of demanded [general permissions](#general-permissions).
+- `options` (_optional_)
+  - `actionRights`: (_optional_) an array of [action rights](#action-rights).
```

Comment:
Hm. I understand the intend but from a "code-perspective" the mentioned `actionRights` is an entry under `options`. So whenever somebody would pass in the argument in code it would look like `{ options: { actionRights: {} } }` from a nesting perspective. I feel if I would level up the bullet point it gives the false impression that there is no nesting in the options. Does that make sense? Would you still suggest to level it up?